fastlane/lanes/release.rb

```diff
@@ -286,19 +297,18 @@
       • Current release version and build code: #{release_version_current} (#{build_code_current}).
       • New release version and build code: #{new_version} (#{build_code_hotfix}).
 
-      Branching from tag: #{previous_version}
+      Branching from #{base_ref_for_hotfix}
     MESSAGE
 
     UI.important(message)
     UI.user_error!('Aborted by user request') unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
 
     # Check tags
     UI.user_error!("Version #{new_version} already exists! Abort!") if git_tag_exists(tag: new_version)
-    UI.user_error!("Version #{previous_version} is not tagged! A hotfix branch cannot be created.") unless git_tag_exists(tag: previous_version)
 
     # Create the hotfix branch
-    UI.message 'Creating hotfix branch...'
-    Fastlane::Helper::GitHelper.create_branch(compute_release_branch_name(options: options, version: new_version), from: previous_version)
+    UI.message "Creating hotfix branch from #{base_ref_for_hotfix}..."
```

Comment:
While we're at it, for consistency:

```suggestion
    UI.message("Creating hotfix branch from #{base_ref_for_hotfix}...")
```

Reply:
Updated on 6e6ab38f5ce32b10e839a466c54d9a073fa633d8 to make the lane more consistent.